src/module.ts

```diff
@@ -121,11 +125,17 @@ export default defineNuxtModule<ModuleOptions>({
           }
         })
       }
+      let swGenerated = false
       nuxt.hook('nitro:init', (nitro) => {
         nitro.hooks.hook('rollup:before', async () => {
+          swGenerated = true
           await resolveVitePluginPWAAPI()?.generateSW()
         })
       })
+      nuxt.hook('close', async () => {
+        if (!swGenerated)
+          await resolveVitePluginPWAAPI()?.generateSW()
+      })
```

Comment:
Was this not being generated before?

Reply:
No

Reply:
When running `nuxt generate`, `rollup:before` hook not being called and so only the client build is generated, missing all html pages from the sw precache manifest.